api/IDBVersionChangeEvent.json

```
@@ -68,6 +68,56 @@
           "deprecated": false
         }
       },
+      "IDBVersionChangeEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBVersionChangeEvent/IDBVersionChangeEvent",
+          "spec_url": "https://w3c.github.io/IndexedDB/#dom-idbversionchangeevent-idbversionchangeevent",
+          "description": "<code>IDBVersionChangeEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
```

```suggestion
              "version_added": "79"
```

Based on
https://mdn-bcd-collector.appspot.com/tests/api/IDBVersionChangeEvent/IDBVersionChangeEvent testing false in Edge 18